tools/peptideshaker/searchgui.xml

```diff
@@ -184,139 +346,534 @@
+				<param name="ms_amanda_max_rank" type="integer" value="5" 
+					label="MS Amanda: Maximum Rank" help="MS Amanda maximum rank" />
+				<param name="ms_amanda_mono" type="boolean" truevalue="1" falsevalue="0" checked="true"
+					label="MS Amanda: Use Monoisotopic Mass Values" help="MS Amanda use monoisotopic mass values" />
+            </when>
+        </conditional>
+		
+-->
+		<!-- TIDE ADVANCED PARAMETERS -->
+        <conditional name="tide">
+            <param name="tide_advanced" type="select" label="TIDE Options">
+                <option value="yes">Advanced</option>
+                <option value="no" selected="True">Default</option>
+            </param>
+            <when value="no" />
+            <when value="yes">
+                <param name="tide_num_ptms" type="integer" value="100"
+                    label="TIDE: Maximum Number of PTMs" help="Set the maximum number of PTMs on peptide to be considered"/>
+                <param name="tide_num_ptms_per_type" type="integer" value="2"
+                    label="TIDE: Maximum Number of PTMs of each Type" help="Set the maximum number of PTMs of each type to be considered"/>
+                <param name="tide_min_pep_length" type="integer" value="6"
+                    label="TIDE: Minimum Peptide Length" help="Set the minimum length of peptide to be considered"/>
+                <param name="tide_max_pep_length" type="integer" value="30"
+                    label="TIDE: Maximum Peptide Length" help="Set the maximum length of peptide to be considered"/>
+                <param name="tide_min_prec_mass" type="float" value="200.0"
+                    label="TIDE: Minimum Precursor Mass" help="Set the minimum precursor mass to be considered"/>
+                <param name="tide_max_prec_mass" type="float" value="7200.0"
+                    label="TIDE: Maximum Precursor Mass" help="Set the maximum precursor mass to be considered"/>
+  	<param name="tide_decoy_format" label="TIDE: Decoy Format" type="select" help="Select the format for generating the decoy sequences">
+			                	<option value="none" selected="True">none</option>
+			                	<option value="shuffle" >shuffle</option>
+			                	<option value="peptide-revers" >peptide-reverse</option>
+			                	<option value="protein-reverse" >protein-reverse</option>
+	</param>
+  	<param name="tide_keep_terminals" label="TIDE: Keep Terminals" type="select" help="Select to keep the terminal amino acids when creating decoys">
+			                	<option value="N" >N</option>
+			                	<option value="C" >C</option>
+			                	<option value="NC" selected="True">NC</option>
+			                	<option value="non" >none</option>
+	</param>
+                <param name="tide_decoy_seed" type="integer" value="1"
+                    label="TIDE: Decoy Seed" help="Set the decoy seed"/>
+                <param name="tide_output_folder" type="text" value="crux-output"
```

Comment:
We kept this, in case users want to generate TIDE output with SearchGUI, without doing the downstream interpretation with PeptideShaker.
I guess this can be left out.